### ISSUE TITLE
What's this other value?

### ISSUE THREAD

**Issue by ladlon** [turtletooth/tahoma2d#323]
I'm a bit rusty on the concept of 'fields' with regards to the Z value.  I'm very used to the concept of the Z axis and all that... but this is the first time I've heard of a unit(?) of 'fields'... So, I'm not really clear on the explanation (although that doesn't prevent me from using the Z axis without problem).

As well, what is the bracketed value after the Z axis one?  (in the screenshot)

For the longest time, I was a bit unsure about whether adjusting these values with regards to the CAMERA was moving the camera in Z or altering the zoom lens focal point.

And, even though I think it's mentioned in the manual, I'm also a little rusty on whether a level's Z value or Stacking Order takes precidence.  Seems like SO is redundant (and maybe even conflicting)... unless you have two or more layers on the same Z level, and you need some means to order them visually.

**Comment by artisteacher** [comment]
<img width="716" alt="Screen Shot 2020-10-08 at 5 15 51 PM" src="https://user-images.githubusercontent.com/21159116/95514452-f295b880-0989-11eb-8047-969c92523cfc.png">

Here's an example:
<img width="639" alt="Screen Shot 2020-10-08 at 5 25 48 PM" src="https://user-images.githubusercontent.com/21159116/95515283-5cfb2880-098b-11eb-8279-d46ca3a7cb6c.png">


Default original size value:
<img width="1033" alt="Screen Shot 2020-10-08 at 5 23 39 PM" src="https://user-images.githubusercontent.com/21159116/95515118-19a0ba00-098b-11eb-9fc5-a49fb787c954.png">

Original size value = Z value:
<img width="1028" alt="Screen Shot 2020-10-08 at 5 23 48 PM" src="https://user-images.githubusercontent.com/21159116/95515166-2cb38a00-098b-11eb-8a51-d511227f7621.png">

**Comment by ladlon** [comment]
Ya, see, that manual has a real talent of making my eyes cross...

It's hard to wrap my head around the heirarchy of the three attributes... and, frankly, why three are needed.

I can see a second value brought into play if there is a conflict and a tie breaker is needed (as in the case of two or more layers being in the same z depth)... but I'm failing to understand the logic of the current system.

Actually, now that I think of it, you don't even need a tie breaker, if you simply use column order as your stacking order.  With just that, no columns can have the same 'level' since one always has to be before the other.  So, now I'm even more confused.

A 'stacking order' attribute would only really seem relevant (and not redundant) if you want to animate the stacking order (normally defined by the column order).  But, then again, why not just give each column a 'layer number' attribute (which is automatically assigned based on the column order... and then by animating/altering that value, the column physically moves to a different position in the actual column order (in the XSheet/Timeline)?  Besides making things far more simple, the 'stacking order' would always be logical, as it would always be based on the actual column order.  In terms of the 3D view, you could give the layer a specific Z depth, but not allow it to move through/past a layer above/below it (...It could just push all stacks above or below it to make room for the layer you are adjusting the Z position for.

Actually, you could just do it solely with Z position, and ignore column order.  Just give each level a Z depth value, and that's all you need.  That Z value could be animated (causing the level to physically move closer/farther from the camera, and would naturally obscure whatever it is in front of... and that way, you don't need stacking order, nor column order.

Either the system is unnecessarily complicated/redundant... or I'm missing some very important aspect of this whole concept.

**Comment by artisteacher** [comment]
I hear you. There are so many "tips" that it can be hard to find what you are looking for. There's just so much there, I'm not sure of the best approach to make the information more accessible.

There may be some redundancy but each process has its benefits. For anyone just starting out, the layer/column order is easy to understand and all most beginners need. If someone has a flat style but wants to swap the order, SO is helpful. It is just a single number to update. It's also the only way to control how an image overlaps itself when using the plastic tool. Z position is necessary when you want to fake a more 3d feel.

**Comment by ladlon** [comment]
I was going with a pure column-based stack... but that plan got messed up when I needed to use an OverFX node simply to funnel several feeds into a single node... and then the input order of the OverFX node seems to take priority.

That explains why the scene would sometimes switch some level's stacking order (visually in the viewport) when I toggled the 'full preview' button thing (as that then includes all the things in the FX tree, including that OverFX node).... or, that was my theory.

Initially, my use of the OverFX node was often to run several nodes (objects, body parts, etc) through a single, shared FX node... but then someone pointed out linked nodes, and so that was no longer necessary.

Just seems to get a bit messy and hard to keep track of when you have all three factors playing on the visual stack order.
